planner/core/task.go

```diff
@@ -728,11 +771,15 @@ func (p *PhysicalStreamAgg) attach2Task(tasks ...task) task {
 
 // GetCost computes cost of stream aggregation considering CPU/memory.
 func (p *PhysicalStreamAgg) GetCost(inputRows float64, isRoot bool) float64 {
+	numAggFunc := len(p.AggFuncs)
+	if numAggFunc == 0 {
+		numAggFunc = 1
```

Comment:
should we use the number of `group by` columns?

Reply:
I also think the number of `group by` columns is useful for cost .  @eurekaka  PTAL